lib/simulator-xcode-6.js

```diff
@@ -658,3 +658,4 @@ for (let [cmd, fn] of _.toPairs(extensions)) {
 }
 
 export default SimulatorXcode6;
+export { ON_BOOT_COMPLETED_EVENT };
```

Comment:
It's best to also add the default to the list

```js
export default SimulatorXcode6;
 +export { SimulatorXcode6, ON_BOOT_COMPLETED_EVENT };
```

So that an import can be like
```js
import { SimulatorXcode6, ON_BOOT_COMPLETED_EVENT };
```
rather than
```js
import { default as SimulatorXcode6, ON_BOOT_COMPLETED_EVENT };
```